src/main/java/SQL_createTable/SQL.java

```diff
@@ -0,0 +1,184 @@
+            stm.executeUpdate(sql);
+            connection.commit();
+
+        } catch (SQLException | RuntimeException e) {
+            System.err.println(e);
+            logger.error(e);
+        } finally {
+            stm.close();
+            connection.close();
+            logger.info("Resources are closed.");
+        }
+    }
+
+    private static void insertValuesIntoTableAuthor() throws SQLException {
+        Connection connection = null;
+        Statement stm = null;
+        try {
+            connection = getConnection();
+            connection.setAutoCommit(false);
+            stm = connection.createStatement();
+            String sql = "INSERT INTO author (id, name ,surname) VALUES (1,'Сергей','Есенин')," +
+                    "(2,'Владимир','Высоцкий')," +
+                    "(3,'Александр','Пушкин')";
+            stm.executeUpdate(sql);
+            connection.commit();
+
+        } catch (SQLException | RuntimeException e) {
+            System.err.println(e);
+            logger.error(e);
+        } finally {
+            stm.close();
+            connection.close();
+            logger.info("Resources are closed.");
+        }
+    }
+
+    private static void insertValuesIntoTableBook() throws SQLException {
+        Connection connection = null;
+        Statement stm = null;
+        try {
+            connection = getConnection();
+            connection.setAutoCommit(false);
+            stm = connection.createStatement();
+            String sql = "INSERT INTO book (id, title, author_id, pages) VALUES (1,'Письмо к женщине',1,100)," +
+                    "(2,'Роман о девочках',2,132)," +
+                    "(3,'Зимнее утро',3,245)";
+            stm.executeUpdate(sql);
+            connection.commit();
+
+        } catch (SQLException | RuntimeException e) {
+            System.err.println(e);
+            logger.error(e);
+        } finally {
+            stm.close();
+            connection.close();
+            logger.info("Resources are closed.");
+        }
+    }
+
+    private static void displayBookPages() throws SQLException {
+        Connection connection = null;
+        Statement stm = null;
+        ResultSet rs = null;
+        try {
+            connection = getConnection();
+            connection.setAutoCommit(false);
+            stm = connection.createStatement();
+            rs = stm.executeQuery("select title AS Book, pages AS Pages from book\n" +
+                    "where pages > 100");
+            while (rs.next()) {
+                String str = rs.getString("Book") + " " + rs.getString("Pages");
+                System.out.println(str);
+            }
+
+        } catch (SQLException | RuntimeException e) {
+            System.err.println(e);
+            logger.error(e);
+        } finally {
+            connection.close();
+            stm.close();
+            rs.close();
```

Comment:
Ideally, resources should be closed in the reverse order of initialization:
what I mean, if we open them in connection -> statement -> resultset
then we should close them as resultset -> statement -. connection